apps/dc_tools/odc/apps/dc_tools/sqs_to_dc.py

```diff
@@ -400,14 +406,15 @@ def cli(
     allow_unsafe,
     record_path,
     region_code_list_uri,
+    absolute,
     queue_name,
     product,
 ):
     """ Iterate through messages on an SQS queue and add them to datacube"""
 
     transform = None
     if stac:
-        transform = stac_transform
+        transform = lambda stat_doc: stac_transform(stat_doc, not absolute)
```

Comment:
actually, maybe explicitly naming second parameter to `stac_transform` would be clearer.